### DISCUSSION TITLE
ubo cli (discussion)

### DISCUSSION THREAD

**Discussion by mehrdadfeller** [ubopod/ubo_app#125] [Ideas]
It might make sense to expose a simple CLI to some stuff on the command line. I am not sure clear on what capability the cli would have or not but some possibilities can be:

- ubocli -screenshot output.png
- ubocli -testall (run some diagnostic script)

Please let me know if you have any suggestions

**Comment by zytegalaxy** [comment]
Now that we have the screen shortcuts and other capabilities that are offered by the GUI, I am going to close this discussion. If necessary, we can reopen or create a new one later.
